index.bs

```diff
@@ -1496,18 +1497,55 @@ Note: Content generally should not provide a visualization of the {{boundsGeomet
 
-A <dfn>view</dfn> corresponds to a display or portion of a display used by an XR device to present imagery to the user. They are used to retrieve all the information necessary to render content that is well aligned to the [=view=]'s physical output properties, including the field of view, eye offset, and other optical properties. [=Views=] may cover overlapping regions of the user's vision. No guarantee is made about the number of [=views=] any XR device uses or their order, nor is the number of [=views=] required to be constant for the duration of an {{XRSession}}.
+A <dfn>view calibration</dfn> corresponds to the set of intrinsics and extrinsics used to translate between a point in the [=XRSession/viewer reference space=]'s [=coordinate system=], and in the [=view calibration/containing object=]'s [=view calibration/screen space=].
 
-A [=view=] has an associated internal <dfn>view offset</dfn>, which is an {{XRRigidTransform}} describing the position and orientation of the [=view=] in the [=XRSession/viewer reference space=]'s [=coordinate system=].
+A [=view calibration=] has a <dfn for="view calibration">containing object</dfn> which is the physical piece of hardware that the [=view calibration=] contains data for.
+
+A [=view calibration=]'s [=view calibration/containing object=] has an associated <dfn for="view calibration">screen space</dfn>, which is described as the 2D plane that this [=view calibration/containing object=] either reads data from or renders data to.
+
+A [=view calibration=] has an associated <dfn for="view calibration">view offset</dfn>, which is an {{XRRigidTransform}} describing the position and orientation of the [=view calibration/containing object=] in the [=XRSession/viewer reference space=]'s [=coordinate system=].
 
 NOTE: There are no constraints on what the view offset might be, and views are allowed to have differing orientations. This can crop up in head-mounted devices with eye displays centered at an angle, and it can also surface itself in more extreme cases like CAVE rendering. Techniques like z-sorting and culling may need to be done per-eye because of this.
 
-A [=view=] has an associated <dfn for="view">projection matrix</dfn> which is a [=matrix=] describing the projection to be used when rendering the [=view=], provided by the underlying XR device. The [=view/projection matrix=] MAY include transformations such as shearing that prevent the projection from being accurately described by a simple frustum.
+A [=view calibration=] has an associated <dfn for="view calibration">projection matrix</dfn> which is a [=matrix=] describing the projection to be used when rendering to the [=view calibration/containing object=] (or the inverse of the projection to use when reading from the object), provided by the underlying XR device. The [=view calibration/projection matrix=] MAY include transformations such as shearing that prevent the projection from being accurately described by a simple frustum.
```

Comment:
"(or the inverse of the projection to use when reading from the object)"

Wait, when we return the projection for a depth view, is it already inverted? If so, it feels a little weird to label in `projectionMatrix`. I kinda wonder if we shouldn't just have a wholly separate `XRDepthView` type at that point that has an explicit `inverseProjectionMatrix`.

If, on the other hand, we return the `projectionMatrix` for the lens that captured the depth map and the developer is intended to invert it I think this approach is still appropriate but the phrasing of this sentence needs to be updated.

Reply:
I was basing this off of my implementation to map between the XRView and "sampling" a point from the depth buffer, where one of these was inverted: https://source.chromium.org/chromium/chromium/src/+/main:device/vr/openxr/android/openxr_depth_sensor_android.cc;drc=a3a53e2b858715bb6f8ea1adbfda034f3fedc0de;l=184

I *think* the `projectionMatrix` works if you are going from screen space to camera space (which might be called "world space")

Reply:
Essentially, I'm pretty sure it's the latter, we provide the matrix to map from the world to the screen space, and if you want to take e.g. a point from the screen and map it back to the world you need to invert the matrix.

Maybe instead of a parenthetical a non-normative note that the inverse of this matrix can be used to "read" values from the screen-space to the "real world"?

Reply:
Proposed note verbiage SGTM!